### DISCUSSION TITLE
[Feedback] Team Plan Demo

### DISCUSSION THREAD

**Discussion by evi-liu** [community/community#68552] [Other Feature Feedback, Questions, & Ideas]
👋 Hey folks, if you tried out the demo for the [GitHub Team plan](https://github.com/pricing), what feedback do you have?

- What did you like about the demo?
- What was confusing or could be improved about the demo?
- If you're looking to upgrade to the Team plan after using the demo, what part of the demo stood out to you?
- If you're not looking to upgrade to the Team plan using the demo, why not and what would convince you?

<img src="https://github.com/community/community/assets/8809849/404096f1-1b80-4133-ab7f-2df4753a5523" width="600" height="200" />

_*Note that at this time, this demo experience is only available to a subset of users. If you have access, the demo can be found at https://github.com/organizations/_[your_free_org]_/billing/plans._

**Comment by sonnees** [comment]
hi, can you rate this comment as an answer? I'm on a mission :v

**Comment by thomagos** [comment]
Overall: I really liked it, as it helped me to see the feature in action. 

Confusing: At first it wasn't very clear to me that the green pulsating dot meant where to look, mainly because it was on a PR and I thought it was a new feature indicating a running pipeline. Starting with another feature first, to get the user used to the green dot, would help.

Upgrade after the demo: No, I would like to see some of the security and compliance features first, like the "Security overview".
